examples/postgres-RDS/README.md

```diff
@@ -0,0 +1,329 @@
+    --engine-version 15.2
+    --master-username <master-username> \
+    --vpc-security-group-ids <security-group-id> \ # Sec group with TCP 5432 inbound rule
+    --master-user-password <db-password>
+
+aws rds wait db-instance-available --db-instance-identifier=<instance-name>
+```
+## Create configmap
+
+Create a configmap that contains information to connect to the RDS DB instance
+
+```
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: dbconfig
+data:
+  region: <region-name> #Region in which instance is to be created
+  instance_name: <instance-name>  # name of the RDS Instance that would be provisioned
+```
+
+## Create secret
+
+Create a secret that contains credentials to connect to the RDS DB instance
+
+```
+apiVersion: v1
+kind: Secret
+metadata:
+  name: dbsecret
+stringData:
+  ########## AWS Key CREDS
+  accessKeyId: 
+  secretAccessKey:
+  ########## Database instance creds
+  postgres_username: <master-username>
+  postgres_password: <db-password> 
+```
+
+**NOTE**
+
+The AWS credentials must have proper permissions to perform operations on RDS
+
+## Installing the PostgreSQL Chart
+To install the chart with the release name `my-release`:
+
+```bash
+$ helm repo add bitnami https://charts.bitnami.com/bitnami
+$ helm repo update
+
+$ helm install my-release --create-namespace --namespace postgres-test bitnami/postgresql
+```
+
+The command deploys PostgreSQL on the Kubernetes cluster in the default configuration.
+
+> **Tip**: List all releases using `helm list`
+
+In case, if you don't have `Kanister` installed already, you can use the following commands to do that.
+Add Kanister Helm repository and install Kanister operator
+```bash
+$ helm repo add kanister https://charts.kanister.io
+$ helm install kanister --namespace kanister --create-namespace kanister/kanister-operator
+```
+
+## Integrating with Kanister
+
+If you have deployed a PostgreSQL application with a name other than `my-release` and a namespace other than `postgres-test`, you need to modify the commands used below to use the correct name and namespace
+
+### Create Profile
+
+Create Profile CR if not created already
+
+```bash
+$ kanctl create profile s3compliant --access-key <aws-access-key-id> \
+	--secret-key <aws-secret-key> \
+	--bucket <s3-bucket-name> --region <region-name> \
+	--namespace postgres-test
+```
+
+**NOTE:**
+
+The command will configure a location where artifacts resulting from Kanister
+data operations such as backup should go. This is stored as a `profiles.cr.kanister.io`
+*CustomResource (CR)* which is then referenced in Kanister ActionSets. Every ActionSet
+requires a Profile reference to complete the action. This CR (`profiles.cr.kanister.io`)
+can be shared between Kanister-enabled application instances.
+
+## Create Docker Image to be used in the Blueprint
+Create the docker image which include awscli, postgresclient and kando (https://docs.kanister.io/tooling.html#kando) . We would provide the image to the blueprint.
```

Comment:
What is `postgresclient`? Do you mean postgres CLI?

Reply:
yeah, I meant `psql`

Reply:
let's mention psql CLI instead of `postgresclient`.

Reply:
I have already modified it to `psql` there.